build.sbt

```diff
@@ -22,8 +22,8 @@ developers := List(
   )
 )
 
-scalaVersion := "2.13.5"
-crossScalaVersions := Seq("2.11.12", "2.12.11", "2.12.12", "2.13.6", "2.13.3", "2.13.4", "2.13.5")
+scalaVersion := "2.13.6"
+crossScalaVersions := Seq("2.11.12", "2.12.11", "2.12.12", "2.12.13", "2.13.3", "2.13.4", "2.13.5", "2.13.6")
```

Comment:
Latest 2.12 was updated to 2.13.6 by https://github.com/sksamuel/scapegoat/pull/523. This keeps the order and support for latest 2.12.
We might want to drop 2.12.11.